src/state/searchState.ts

```diff
@@ -92,20 +92,38 @@ export class SearchState {
 
       let lineIdx = lineToStartAt;
 
+      let numNewLines = (searchRE.match(/\\n/g) || []).length;
       // Start at the current line and wrap the document if we hit the end.
       outer:
       do {
-        const line = TextEditor.getLineAt(new Position(lineIdx, 0)).text;
+        let rangeStart = new Position(lineIdx, 0);
```

Comment:
Minor I know, but can we have a convention of using `const` when the variable doesn't change?